src/Workspaces/Core/Portable/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class CodeStyleHelpers
+    {
+        public static CodeStyleOption<bool> ParseEditorConfigCodeStyleOption(string arg)
+        {
+            var args = arg.Split(':');
+            bool isEnabled = false;
+            if (args.Length != 2)
+            {
+                if (arg.Length == 1)
+                {
+                    if (bool.TryParse(args[0].Trim(), out isEnabled) && !isEnabled)
+                    {
+                        return new CodeStyleOption<bool>(value: isEnabled, notification: NotificationOption.None);
+                    }
+                    else
+                    {
+                        return CodeStyleOption<bool>.Default;
+                    }
+                }
+                return CodeStyleOption<bool>.Default;
+            }
+
+            if (!bool.TryParse(args[0].Trim(), out isEnabled))
+            {
+                return CodeStyleOption<bool>.Default;
+            }
+
+            switch (args[1].Trim())
+            {
+                case "none": return new CodeStyleOption<bool>(value: isEnabled, notification: NotificationOption.None);
```

Comment:
Still not sure I like "none" because it's confusing to me if that's the severity or the value. 'silent' perhaps?